<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kitchen</class>
 <widget class="QDialog" name="kitchen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: Silver;</string>
  </property>
  <widget class="QPushButton" name="logout_kitchen_Btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	font-size: 50px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>â†©</string>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="LAMP_Btn">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>0</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>LAMP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Fri_Btn">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>0</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Fridge</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>800</width>
     <height>550</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #d9e7ff, stop:1 #e3ffe7);
</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="Fri_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Off_Fri_Btn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>460</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	border-radius: 10px;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="On_Fri_Btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>460</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="Off_GAS_Btn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>433</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	border-radius: 10px;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="up_GAS_Btn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>330</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>â–²</string>
     </property>
    </widget>
    <widget class="QPushButton" name="On_GAS_Btn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>433</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	border-radius: 10px;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QLabel" name="GAS_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>67</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
color: rgb(0, 0, 0);
font-size: 30px;</string>
     </property>
     <property name="text">
      <string>GAS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="down_GAS_Btn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>360</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>â–¼</string>
     </property>
    </widget>
    <widget class="QLabel" name="Temp_label">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>220</y>
       <width>140</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
font-size: 20px;
font-weight: 600;
border-style: solid;
border-width: 2px;
border-color: #9999aa;
</string>
     </property>
     <property name="text">
      <string>Temp</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_khi_gas_img">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>80</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 0);</string>
     </property>
     <property name="text">
      <string>Temp</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="On_LAMP_yellow_Btn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>210</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	border-radius: 10px;
	font-size: 20px;
	font-weight: 600;
	background-color: rgb(245, 236, 77);
}

QPushButton:enabled {
		background-color: rgb(245, 236, 77);

		color: white;
}

QPushButton:pressed {
		background-color: rgb(245, 236, 77);
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: rgb(255, 255, 255);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>130</y>
       <width>60</width>
       <height>200</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar
{
    background: #74c8ff;
	color: rgb(255, 255, 255);
    border-style: outset;
    border-width: 2px;
	border-color: #9999aa;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
    border-radius: 7px;
	font-size: 20px;
    text-align: center;
}
QProgressBar::chunk
{
    background-color: rgb(170, 0, 0);
    border-radius: 7px;

}
 /*background-color: #010327;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QLabel" name="LAMP_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>300</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;
color: rgb(0, 0, 0);
font-size: 30px;</string>
     </property>
     <property name="text">
      <string>Lamp</string>
     </property>
    </widget>
    <widget class="QSlider" name="PW_LAMP_hS">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>390</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">    background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;

</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="On_LAMP_Btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	border-radius: 10px;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="On_LAMP_red_Btn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>160</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	border-radius: 10px;
	font-size: 20px;
	font-weight: 600;
	background-color: rgb(236, 56, 50);
}

QPushButton:enabled {
		background-color: rgb(236, 56, 50);
		color: white;
}

QPushButton:pressed {
		background-color: rgb(236, 56, 50);
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: rgb(255, 255, 255);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="On_LAMP_blue_Btn">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	border-radius: 10px;
	font-size: 20px;
	font-weight: 600;
	background-color: rgb(73, 185, 234);
}

QPushButton:enabled {
		background-color: rgb(73, 185, 234);
		color: white;
}

QPushButton:pressed {
		background-color: rgb(73, 185, 234);
		color: #fffffe;
}


QPushButton:hover:!pressed {
		background-color: rgb(255, 255, 255);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Off_LAMP_Btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>420</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	border-radius: 10px;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="upgrate_kitchen_Btn">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>0</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Upgrate</string>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="GAS_Btn">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>0</y>
     <width>210</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	color: white;
	font-size: 20px;
	font-weight: 600;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>GAS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
